### DISCUSSION TITLE
Allow progress tracking of scenario objectives in the park menu

### DISCUSSION THREAD

**Discussion by JonasKlose** [OpenRCT2/OpenRCT2#15774] [Feature requests]
I've had a problem with the RCT2 scenario "Ghost Town" as I wasn't sure how the rules of the game worked.
The requirement was to build 10 roller coasters of different types that all were 1200 units long.
However I thought that doing multiple laps would increase the length of the rides like it does with other stats.

But I wasn't sure and saw no indicator if I managed to do any progress on the scenario or not.
So I looked into the source to find the exact requirements.

The proposal is to include a progress tracker in the main park ui.

OpenRCT2, v3.2.0

**Comment by MakkeW** [comment]
I was thinking of handling this issue, but I'll probably take care of #13792 and refactor the park window first as I figure that's where the progress will be shown.
